analysis/question_2/top_suicide_rate.py

```diff
@@ -0,0 +1,15 @@
+import os
+import pandas as pd
+
+# Getting Data
+suicide_rates_data_filepath = os.path.join("raw_data", "crude_suicide_rates.csv")
+suicide_rates_dataframe = pd.read_csv(suicide_rates_data_filepath)
+
+# Filter youth data only
+youth_suicide_rates_dataframe = suicide_rates_dataframe[suicide_rates_dataframe["Sex"] == " Both sexes"][["Country", " 10to19"]]
```

Comment:
Is there one spacing before the column names "10to19" and "Both sexes" in the original CSV ?